server/routes/courtRegister/courtRegisterController.ts

```diff
@@ -36,13 +38,25 @@ export default class CourtRegisterController {
 
   async showAllCourtsPaged(req: Request, res: Response): Promise<void> {
     const page = parseInt(req.query.page as string, 10) || 1
-    const courtsPage = await this.courtRegisterService.getPageOfCourts(context(res), page - 1, 40)
+    const filter = this.parseFilter(req)
+    const courtsPage = await this.courtRegisterService.getPageOfCourts(context(res), page - 1, 40, filter)
 
-    const view = new AllCourtsPagedView(courtsPage)
+    const view = new AllCourtsPagedView(courtsPage, filter)
 
     res.render('pages/court-register/allCourtsPaged', view.renderArgs)
   }
 
+  parseFilter(req: Request): AllCourtsFilter {
+    const activeString: string = req.query.active as string
+    let active = null
```

Comment:
this is a bit messy having a mutable variable here, maybe pull out into a `parseBoolean` method?